### ISSUE TITLE
Exception when using coffea.lookup_tools.extractor on DeepJet CSV file

### ISSUE THREAD

**Issue by jrueb** [scikit-hep/coffea#206]
**Describe the bug**
When trying to open the DeepJet CSV file of the CMS BTV with a coffea extractor, an exception is raised.

**To Reproduce**
To reproduce, one can use the following
```
extractor = coffea.lookup_tools.extractor()
wgts = "* * DeepJet_102XSF_WP_V1.csv"
extractor.add_weight_sets([wgts])
```
It raises an IndexError. The parser seems to expect a semicolon in the header, while there is none for the DeepJet CSV files. The semicolon is present in DeepCSV CSV files.

**Comment by lgray** [comment]
@jrueb Can you give a url to that csv file? I'll cook a fix.

**Comment by jrueb** [comment]
The CSV file can be found here https://twiki.cern.ch/twiki/pub/CMS/BtagRecommendation102X/DeepJet_102XSF_WP_V1.csv

**Comment by lgray** [comment]
Thanks, I'll patch it up in a bit.

**Comment by lgray** [comment]
@jrueb This is fixed in coffea `0.6.18` which will is available now in master from github or on PyPI in about an hour.
